react/src/components/tabs/components/SprkTabsButton/SprkTabsButton.js

```diff
@@ -20,18 +20,21 @@ class SprkTabsButton extends Component {
       tabBtnChildren,
       tabBtnId,
       tabBtnAddClasses,
+      tabBtnAdditionalClasses,
       ariaControls,
       ariaSelected,
       isActive,
       onTabClick,
       tabBtnDataId,
       tabBtnAnalytics,
+      tabBtnAnalyticsString,
     } = this.props;
     return (
       <button
         className={classnames(
           'sprk-c-Tabs__button',
-          tabBtnAddClasses,
+          tabBtnAdditionalClasses || tabBtnAddClasses,
+          // eslint-disable-next-line prettier/prettier
```

Comment:
Just curious - What's prettier's problem with this line?

Reply:
it keeps trying to remove the { and } and it breaks `classnames`.